src/grammar/index.ts

```diff
@@ -569,6 +569,11 @@ const parseStatementCondition =
         loc,
     }: $ast.StatementCondition): Handler<Ast.StatementCondition> =>
     (ctx) => {
+        if (typeof condition === "undefined") {
+            ctx.err.expectedExpression("after if keyword")(loc);
+            condition = trueLiteral;
```

Comment:
I would prefer if you found another solution here. This looks like a horrible hack :)

Reply:
> This looks like a horrible hack :)
yes